src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,35 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
+import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
```

Comment:
it's not okay to add the spaces this way, better to use the margins and padding, for example:
`<span class='CommentInfo__title-by'>by</span>`

please check other places